### PR TITLE
sdl2 update to 2.0.10

### PR DESCRIPTION
**Pull request by Restia666Ashdoll**
Update sdl2 from 2.0.7 to 2.0.10

### CODE REVIEW

**Comment by Ash258** [comment]
- Add checkver 
- Why this manifest contains development libraries instead of runtime libraries?

**Comment by Ash258** [comment]
Did I tell something about autoupdate? Remove it.

**Comment by Restia666Ashdoll** [comment]
The runtime libraries only contain a dll file that you use with other applications. You have to compile it yourself with Visual C++

File: bucket/sdl2.json [hunk 1]
```diff
@@ -73,5 +76,9 @@
             "$was_in_path, $newpath = strip_path $env:CMAKE_PREFIX_PATH $cmdir",
             "if($was_in_path) { $env:CMAKE_PREFIX_PATH = $newpath }"
         ]
-    }
+    },
+	"checkver": {
```
Comment (Ash258) [inline bucket/sdl2.json]:
```suggestion
    "checkver": {
		    "url": "https://www.libsdl.org/download-2.0.php",
		    "regex": "SDL version ([\\d.]+)"
		}
```

**Comment by linsui** [comment]
Please format your manifest. Use 4 spaces instead of tab.

**Comment by Restia666Ashdoll** [comment]
Its not really my manifest. I formatted the parts I added with 4 spaces.

File: bucket/sdl2.json [hunk 2]
```diff
@@ -73,5 +76,10 @@
             "$was_in_path, $newpath = strip_path $env:CMAKE_PREFIX_PATH $cmdir",
             "if($was_in_path) { $env:CMAKE_PREFIX_PATH = $newpath }"
         ]
-    }
+    },
+    "checkver": {
+	    "url": "https://www.libsdl.org/download-2.0.php",
+		"regex": "SDL version ([\\d.]+)"
+		}
```
Comment (linsui) [inline bucket/sdl2.json]:
```suggestion
```

File: bucket/sdl2.json [hunk 3]
```diff
@@ -73,5 +76,10 @@
             "$was_in_path, $newpath = strip_path $env:CMAKE_PREFIX_PATH $cmdir",
             "if($was_in_path) { $env:CMAKE_PREFIX_PATH = $newpath }"
         ]
-    }
+    },
+    "checkver": {
+	    "url": "https://www.libsdl.org/download-2.0.php",
+		"regex": "SDL version ([\\d.]+)"
```
Comment (linsui) [inline bucket/sdl2.json]:
```suggestion
        "regex": "SDL version ([\\d.]+)"
```

File: bucket/sdl2.json [hunk 4]
```diff
@@ -73,5 +76,10 @@
             "$was_in_path, $newpath = strip_path $env:CMAKE_PREFIX_PATH $cmdir",
             "if($was_in_path) { $env:CMAKE_PREFIX_PATH = $newpath }"
         ]
-    }
+    },
+    "checkver": {
+	    "url": "https://www.libsdl.org/download-2.0.php",
```
Comment (linsui) [inline bucket/sdl2.json]:
```suggestion
        "url": "https://www.libsdl.org/download-2.0.php",
```

File: bucket/sdl2.json [hunk 5]
```diff
@@ -73,5 +76,10 @@
...
+		"regex": "SDL version ([\\d.]+)"
+		}
+	}
```
Comment (linsui) [inline bucket/sdl2.json]:
```suggestion
    }
```

**Pull request merged**